ethereum_history_api/oracles/src/noir/noir_js/encode.ts

```diff
@@ -18,6 +18,13 @@ export function encodeHexStringToArray(value: string): Uint8Array {
   return new Uint8Array(arr);
 }
 
+export const encodeNullable = (value: string | null): string =>
```

Comment:
A bit complicated syntax, maybe we can change it to export function ...?

Reply:
changed